frontend_app_kanban/src/component/Kanban.jsx

```diff
@@ -55,11 +57,11 @@ export class Kanban extends React.Component {
       autoCompleteItemList: [],
       board: { columns: [] },
       boardState: justCreated ? BOARD_STATE.LOADED : BOARD_STATE.LOADING,
+      boardInitiallyLoaded: justCreated,
```

Comment:
is it mandatory to have it is a separated variable or could it be another BOARD_STATE?
like BOARD_STATE.INIT

Reply:
I did this, thanks